helm/cert-operator/templates/pss-exceptions.yaml

```diff
@@ -0,0 +1,20 @@
+kind: PolicyException
+metadata:
+  name: {{ include "resource.default.name" . }}-exceptions
```

Comment:
nitpick: This is already a `PolicyException`, adding the `exceptions` suffix sounds redundant to me